operator/gitops/argocd/pipeline-service/tekton-results/base/watcher-logging.yaml

```diff
@@ -0,0 +1,14 @@
+# Adjust logging level of the watcher
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-results-config-logging
+  namespace: tekton-pipelines
+data:
+  loglevel.controller: info
+  loglevel.watcher: info
```

Comment:
the doc at https://github.com/openshift-pipelines/pipeline-service/blob/main/developer/config.yaml refs `loglevel.controller`, but in looking at `tekton-results-config-logging` I also saw `loglevel.watcher`

I decided to include both to deal with any pivots in either upstream results or knative in the future.